src/Components/UserInfo.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import UserRepoInfo from './UserRepoInfo.jsx';
+
+class UserInfo extends React.Component {
+    constructor(props){
```

Comment:
czy ten konstruktor jest potrzebny?

Reply:
hmm, najprościej mówiąc  jest to przyzwyczajenie jeszcze z kursu